recipes/sensor.rb

```diff
@@ -58,6 +58,17 @@
   runit_service 'mss' do
     action :enable
   end
+
+  creds = data_bag_item('credentials', 'metaflows')
+
+  template '/etc/rc0.d/K01delete-metaflows-sensor' do
```

Comment:
Noah and I just learned that on halt K is run before S. And in the log upload stuff in heph, if it was S, it ran after things like networks were gone.